src/figuresClasses.ts

```diff
@@ -2,10 +1,76 @@
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('should check whether it is a triangle');
+    }
+
+    const bigSide: number = Math.max(a, b, c);
+
+    if (a + b + c - bigSide <= bigSide) {
+      throw new Error("sides 1, 2 and 3 can't form a triangle");
```

Comment:
The error message 'sides 1, 2 and 3 can't form a triangle' could be more descriptive. Consider something like 'The provided side lengths do not satisfy the triangle inequality theorem.'